tests/manifests/default/controller_def_1.3.3.yaml

```diff
@@ -14,29 +14,25 @@ data:
           kind: ServiceAccount
           metadata:
             labels:
-              app.kubernetes.io/instance: '{{ .InstanceID }}'
```

Comment:
Removed these "instance" labels from the 1.3.3 manifest - this effectively enables the e2e test to duplicate the issue we had before in the clusters - moving from one version which did not have the labels to a version which did